spec/features/user/signup_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe "user" do
+  it "can sign up with email" do
+    visit root_path
+    click_link_or_button('Sign up')
+    expect(current_path).to eq(new_user_path)
```

Comment:
This isn't that necessary. You can leave it, but it's probably implied since the user clicked a button to sign up. It would fail on the `fill_in` part if we weren't on that path.